atom/browser/browser.h

```diff
@@ -134,6 +134,15 @@ class Browser : public WindowListObserver {
 
   // Set docks' icon.
   void DockSetIcon(const gfx::Image& image);
+
+  // Get login item status of app
+  v8::Local<v8::Value> GetLoginItemStatus(mate::Arguments* args);
```

Comment:
The `atom::Browser` class should not interactive with V8 directly, APIs added here are also meant to be used by non-V8 parts of code. When type conversions and optional arguments are needed, we usually add a wrapper in `api::App`.